gsr_booking/serializers.py

```diff
@@ -6,21 +6,23 @@
 User = get_user_model()
 
 
-class GroupBookingRequestSerializer(serializers.Serializer):
+class GroupRoomBookingRequestSerializer(serializers.Serializer):
     lid = serializers.IntegerField()
     room = serializers.IntegerField()
 
     start = serializers.DateTimeField()
     end = serializers.DateTimeField()
 
-    group_id = serializers.IntegerField()
-
     is_wharton = serializers.SerializerMethodField()
 
     def get_is_wharton(self, obj):
         return obj["lid"] == 1
 
 
+class GroupBookingRequestSerializer(serializers.Serializer):
+    room_bookings = serializers.ListField(child=GroupRoomBookingRequestSerializer())
```

Comment:
```suggestion
    room_bookings = GroupRoomBookingRequestSerializer(many=True)
```
I think this will work in the same way, and it's a bit less verbose. It also matches the ModelSerializer format a bit more for if/when we decide to save all of this to the DB.